### DISCUSSION TITLE
Use sub folder to now/node

### DISCUSSION THREAD

**Discussion by devalexandre** [vercel/vercel#4304] [General]
I try use sub folder with now node, but when I made deploy, in server it list my files.
if I use a self index in root, work.
```
{
  "version": 2,
  "name": "express-now",
  "builds": [
    {"src": "./www/index.js", "use": "@now/node"}
  ],
  "routes": [
    { "src": "/(.*)", "dest": "./www/index.js" }
  ],
  "env": {
    "VERSION": "@version"
  }
}

```

**Comment by coetry** [comment]
Hi @devalexandre can you provide us with a small example so we can reproduce?

**Reply by devalexandre** [reply]
Hi , a git

https://github.com/devalexandre/preact-now

is preact but use SSR
